scripts/setup_qemu_optee.sh

```diff
@@ -254,6 +305,26 @@ $DEV_PATH/build_linux.sh
 # Save kernel version for later use
 export KERNEL_VERSION=`cd $DST_KERNEL && make kernelversion`
 
+
+################################################################################
+# Generate the build script for SE API test                                    #
+################################################################################
+if [ -n "$WITH_SE_API_TEST" ]; then
+
+cat > ${DEV_PATH}/build_se_api_test.sh << EOF
+#!/bin/sh
+
+(cd se_api_test && \
```

Comment:
Line 317-320 needs double backslash at the end (escaped) of the line otherwise they will just disappear the the build script will just be a long line here.